src/F_LSM6DS3.cpp

```diff
@@ -7,16 +7,16 @@ int LSM6DS3::init(calData cal, uint8_t address)
 	//initialize address variable and calibration data.
 	IMUAddress = address;
 
-	if (cal.valid == false) 
+	if (cal.valid == false)
 	{
 		calibration = { 0 };
 	}
 	else
 	{
 		calibration = cal;
 	}
-	
-	uint8_t IMUWhoAmI = checkReady(IMUAddress, 500);
```

Comment:
The problem this is fixing is that 500 is outside the bounds of `uint8_t`. When compiling, I get the warning that this is being implicitly cast to 254 (the max value of uint8_t).

The same code in the `F_LSM6DS3.cpp` file only waits 100ms, and that should be fine here too.